File: tests/fixtures/code_fuzz/directives.py
def publish(bundle: str) -> bool:
    """@codeatlas-fuzz deny: publishes to the real artifact registry"""
    return bool(bundle)


def stale_allow(bundle: str) -> bool:
    """@codeatlas-fuzz allow: stale comments may not grant authority"""
    return bool(bundle)

File: tests/fixtures/code_fuzz/directives.rs
pub trait ArtifactPublisher {
    /// @codeatlas-fuzz deny: publishes to the real artifact registry
    fn publish(&self, bundle: String) -> bool;

    /// @codeatlas-fuzz allow: stale comments may not grant authority
    fn stale_allow(&self, bundle: String) -> bool;
}
